tests/test_tokenization_byt5.py

```diff
@@ -56,6 +56,17 @@ def test_eos_treatment(self):
         batch_without_eos_added = tokenizer(["hi", "I went to the gym", ""])
         self.assertListEqual(batch_with_eos_added["input_ids"], batch_without_eos_added["input_ids"])
 
+    def test_multibytes_char(self):
```

Comment:
Can we also add a test that makes sure that `tokenizer.decode(tokenizer.encode(text)) == text` ?